README.md

```diff
@@ -63,7 +58,7 @@ The trigger is designed hierarchically. The minimal (extremely simplified) flow:
   <img src="docs/minimal.svg" />
 </p>
 
-<p>&nbsp; The channel-level information in the form of Trigger Primitives is merged locally (at the scale of one TDAQ unit, such as an APA or CRP) into a cluster representing some type of Trigger Activity. Multiple Trigger Activity outputs can then be merged into a single Trigger Candidate that may include information spanning multiple TDAQ units or multiple slices of time (say, many APAs or many seconds) for a single system. Finally, multiple Trigger Candidates can be merged across systems (e.g. TPC and PDS) in the MLT.</p>
+The hits-on-wire, in the form of Trigger Primitives, are merged at the scale of one TDAQ unit (such as an APA or CRP) into a cluster representing some type of Trigger Activity. Multiple Trigger Activity outputs can then be merged into a single Trigger Candidate that may include information spanning multiple TDAQ units or multiple slices of time (say, many APAs or many seconds) for a single system. Finally, multiple Trigger Candidates can be merged across systems (e.g. TPC and PDS) in the MLT into a Trigger Decision, a request to save the data.
```

Comment:
Maybe change "hits-on-wire" to "hits-on-channel". The CRPs use copper strips instead of wires for their channels.

Reply:
Agreed & done. Found one more instance of using "wire" rather than "channel", so changed that too.